src/glm_aliasmodel.c

```diff
@@ -489,10 +489,19 @@ static void GLM_RenderPreparedEntities(aliasmodel_draw_type_t type)
 	}
 
 	if (type == aliasmodel_draw_std && alias_draw_instructions[aliasmodel_draw_outlines].num_calls) {
+        extern cvar_t gl_outline_color_model, gl_outline_scale_model;
+
```

Comment:
Should add .editorconfig or .clang-format to the repo to avoid silly indentation issues like this.